src/web/app/pages-instructor/instructor-session-result-page/instructor-session-no-response-panel.component.ts

```diff
@@ -57,6 +69,34 @@ export class InstructorSessionNoResponsePanelComponent implements OnInit, OnChan
     }
   }
 
+  openSendReminderModal(event: any): void {
+    event.stopPropagation();
+
+    const courseId: string = this.session.courseId;
+    const feedbackSessionName: string = this.session.feedbackSessionName;
+
+    const giverEmails: string[] = this.noResponseStudents.map((student: Student) => student.email);
+    const giverSet: Set<string> = new Set(giverEmails);
+
+    const modalRef: NgbModalRef = this.modalService.open(SendRemindersToStudentModalComponent);
+    modalRef.componentInstance.courseId = courseId;
+    modalRef.componentInstance.feedbackSessionName = feedbackSessionName;
+    modalRef.componentInstance.studentListInfoTableRowModels
+      = this.allStudents.map((student: Student) => ({
+        email: student.email,
+        name: student.name,
+        teamName: student.teamName,
+        sectionName: student.sectionName,
+
+        hasSubmittedSession: !giverSet.has(student.email),
+        isSelected: true,
```

Comment:
I suppose only those who do not answer the session should be selected?

Reply:
Apology for the misleading hint :P

Reply:
Alright, in that case `select all respondents` at the top is not selected by default. Only those with `Submitted?` status as **No** will be selected by default. Is it the intended behaviour?

Reply:
Yup. By clicking button, you just want to send reminder to all nonResponseStudents.